models/grouped_assignments.py

```diff
@@ -8,13 +8,16 @@ class AssignmentGrade(object):
     def __init__(self, released, score, projected, assignment_score, assignment_id, assignment_name, grade_record, row):
         self.released = released
         self.score = score
-        self.projected = projected 
-        self.assignment_score = assignment_score
+        self.projected = projected
+        try: assignment_score:
```

Comment:
This line looks like a syntax error.  `try: foo:`  the `foo:` part is definitely wrong.